features/anchor-positioning.yml.dist

```diff
@@ -148,6 +145,7 @@ compat_features:
   - css.properties.place-items.anchor-center
   - css.properties.place-self.anchor-center
   - css.properties.position-anchor
+  - css.properties.position-anchor.auto
```

Comment:
This is from https://github.com/mdn/browser-compat-data/pull/23254 and we'll have to exclude it.

Reply:
Done with cd5ee6f.